### PR TITLE
Removing Lds_hash from Duplicate Participation API Response

### PR DESCRIPTION
**Pull request by ccaprio97**
## What’s changing?

dupp part API endpoint find_matches will no longer return lds_hash

## Why?

NAC-940

## This PR has:

- [X] **Commented source code** (_required on public classes and methods, and elsewhere as appropriate_)

- [ ] **Developer documentation** (_for new build/test/API changes or complex portions of the system_)

- [X] **Automated unit tests** (_to maintain or increase level of code coverage_)

- [ ] **Changes to IAC** (_add any deployment steps that require manual intervention to the draft release notes_)

## Anything else?
image
image

### CODE REVIEW

**Review by dborgen2 — APPROVED** [review]
LGTM

**Comment by ccaprio97** [comment]
@bgfoshay wanted to get your thoughts after andrews message before merging.

File: match/src/Piipan.Match/Piipan.Match.Api/Models/ParticipantMatch.cs [hunk 1]
```diff
@@ -15,6 +15,7 @@ public class ParticipantMatch : IParticipant
 
         [JsonProperty("lds_hash",
             NullValueHandling = NullValueHandling.Ignore)]
+        [JsonIgnore]
```
Comment (bgfoshay) [inline match/src/Piipan.Match/Piipan.Match.Api/Models/ParticipantMatch.cs:18]: Question: Could you add a test to ParticipantTests.cs in Piipan.Match.Api.Tests that verifies that lds_hash is ignored?

Comment (ccaprio97) [reply match/src/Piipan.Match/Piipan.Match.Api/Models/ParticipantMatch.cs:18]: done

**Review by bgfoshay — APPROVED** [review]
LGTM

**Pull request merged**